spec/features/books/destroy_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe "As a visitor", type: :feature do
+  describe "when I click the delete button on an book's show page" do
+    before(:each) do
+      @book_1 = Book.create!(title: "The Frozen Deep", page_count: 106, year_published: 1874, thumbnail: "https://images.gr-assets.com/books/1328728986l/1009218.jpg")
+      @book_2 = Book.create!(title: "To Kill a Mockingbird", page_count: 281, year_published: 1960, thumbnail: "https://upload.wikimedia.org/wikipedia/en/7/79/To_Kill_a_Mockingbird.JPG")
+      @author_1 = @book_1.authors.create!(name: "Wilkie Collins")
```

Comment:
These author Ivars are not used and can be cut.

Reply:
Done -- thanks for catching that